version/version.go

```diff
@@ -0,0 +1,6 @@
+package version
+
+var (
+	// ProviderVersion is set during the release process to the release version of the binary
+	ProviderVersion = "dev"
+)
```

Comment:
You may consider to use below popular way to popular version.go (it can be moved out of source control) during building time:
[https://github.com/git/git/blob/master/GIT-VERSION-GEN](https://github.com/git/git/blob/master/GIT-VERSION-GEN)

Its format looks like git-tag.lastcommitID, which contains enough information for us to track.